specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/types.json

```diff
@@ -84,6 +84,11 @@
       "description": "Common fields that are returned in the response for all Azure Resource Manager resources.",
       "type": "object",
       "properties": {
+        "type": {
+          "description": "The type of the resource. E.g. \"Microsoft.Compute/virtualMachines\" or \"Microsoft.Storage/storageAccounts\".",
+          "type": "string",
+          "readOnly": true
+        },
         "id": {
           "description": "Fully qualified resource ID for the resource. Ex - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}.",
```

Comment:
Should we call this as ResourceId or ResourceUri instead? If we have been using Resource ID then let's continue with the same

Reply:
This should stay as Resource ID (and this actually comes from ARM's definition of a resource).